webdevbroz/pomodoro-app
# Add test suite

## Goal
We want to add testing to our application and have decided on using Jest. Some benefits include:

- simple to no config is necessary, which I think works well for our first application. We don't want to over complicate things.
- Snapshot testing is useful since our application heavily relies on UI usability. We can compare any changes that are made.
- And more!

Related issue: https://github.com/webdevbroz/pomodoro-app/issues/4

## How to test

- Pull this branch and run `yarn install`
- run `yarn test`, and make sure the test passes.

## Screenshots
![image](https://github.com/webdevbroz/pomodoro-app/assets/27636896/25ec9713-b353-47c7-a249-9ca88234888a)
